CHANGELOG.md

```diff
@@ -1,3 +1,51 @@
+# 0.0.118 - Oct XXX, 2023 - "Just the Twelve Sinks"
+
+## API Updates
+ * BOLT12 sending and receiving is now supported as an alpha feature. You may
+   run into unexpected issues and will need to have a direct connection with
+   onion message peer targets in order to exchange messages. We are seeking
+   feedback from early testers (#2578, #2039).
+ * `ConfirmationTarget` has been rewritten to provide information about the
```

Comment:
Could be good to add a migration guide saying which old values map to which new values.

should be

OnChainSweep = HighPriority
MaxAllowedNonAnchorChannelRemoteFee = max(25 * 250, HighPriority * 10)
MinAllowedAnchorChannelRemoteFee = MempoolMinimum
MinAllowedNonAnchorChannelRemoteFee = Background - 250
AnchorChannelFee = Background
NonAnchorChannelFee = Normal
ChannelCloseMinimum = Background